tests/virt_autotest/host_upgrade_base.pm

```diff
@@ -8,7 +8,8 @@
 # without any warranty.
 #
 package host_upgrade_base;
-# Summary: virt_autotest: the initial version of virtualization automation test in openqa, with kvm support fully, xen support not done yet
+# Summary: virt_autotest: The basic libary only for host upgrade test.
+#                         - get_test_name_prefix : Provids the prefix name of test script
```

Comment:
I added some comments about existing issue of ipmi and current project target. I do not know what info do you want to know?